### DISCUSSION TITLE
RSS Feed on Homepage

### DISCUSSION THREAD

**Discussion by jgonzales20** [zen-browser/desktop#3157] [Ideas]
Ability to add RSS feeds to the home page of the browser, this would display similar to stories/news articles.

**Comment by theoneand33** [comment]
This feature reminds me of Vivaldi

**Comment by mr-cheffy** [comment]
Done 👍🏽

**Reply by jgonzales20** [reply]
> Done 👍🏽 

How does one use this feature?
